### DISCUSSION TITLE
Should Synergy sponsor the Deskflow macOS developer code sign certificate?

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#8008] [Polls]
Synergy is the official sponsor of Deskflow.

The macOS bundle is unsigned, so installing the CI-built bundle is cumbersome (you have to [override the macOS security](https://github.com/deskflow/deskflow/wiki/Workarounds#macOS-accessibility-permissions) to make it work). As a result, the recommended solution is to use our Homebrew tap.

If we signed the app bundle using an Apple-issued certificate, then it would be more convenient for testing CI builds.

The cost is currently $99/year (for an Apple Developer account). 

Please add a comment below to add context to your vote.

**Comment by nbolton** [comment]
I'm curious why someone would vote 'no'.

**Comment by nbolton** [comment]
We do plan to do this, but will require some setup; we'll need to setup as a new developer on Apple.

However, I wonder if this is redundant now since `brew` self-signs.

**Reply by sithlord48** [reply]
self signing has some limits. I think this is part of why we have issues w/ some entitlements.
